### DISCUSSION TITLE
doesn't work metadata in use client page

### DISCUSSION THREAD

**Discussion by l1mindr** [vercel/next.js#51420] [App Router]
hi
I'm using nextjs 13.4.5
why doesn't work metadata when use client in page?

**Comment by SamKomesarook** [comment]
Metadata can be added to client components, but they have to be [defined in a layout file](https://nextjs.org/docs/app/building-your-application/optimizing/metadata#static-metadata). Hopefully this gets [addressed](https://github.com/vercel/next.js/discussions/50647) at some point though...
